capport_service/tests/sql.rs

```diff
@@ -0,0 +1,127 @@
+use capport_service::service::sql::HasSqlClient;
+use capport_service::{
+    context::service::{DefaultSvcConfig, DefaultSvcDistributor},
+    service::sql::{SqlClient, SqlClientConfig},
+};
+use serde::{Deserialize, Serialize};
+use sqlx::Postgres;
+
+const SQL_URI: &str = "postgres://postgres:postgres@localhost/csdb";
```

Comment:
Can you make this to be read from `.env` file, and provide the sample .env here thanks
(remember to add the env to gitignore)